bitcoin/fetch_methods.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('@alexbosworth/node-fetch');
+
+//fetch result from method
+module.exports = async ({body, contentType, method, url}) => {
+  
+  try {
+    const response = await fetch(url, {
```

Comment:
for json rpc I think you need to increment an id or something

Reply:
[](https://developer.bitcoin.org/reference/rpc/getrawtransaction.html#argument-2-verbose)Argument #2 - verbose
Type: boolean, optional, default=false

If false, return a string, otherwise return a json object

Is this what you mean? If yes, it defaults to false so already returns json.

Reply:
Oops, Need to set it to true for json.

Reply:
there is something in the POSt request that has to be incremented I think

Reply:
I don't see anything in the docs, not sure. I can look into it.